Prettify the App

evan-m-jackson opened the pull request:
- Add `components` directory
- Remove default Next.js design and begin creating main page design

evan-m-jackson committed:
Prettify the App

- Add `components` directory
- Remove default Next.js design and begin creating main page design

mackenziequinnjetton commented on frontend/__tests__/components/MatchList.test.tsx:
`screen.debug` statement should be removed.

mackenziequinnjetton reviewed (approved):
LGTM, great job!

evan-m-jackson committed:
Prettify the App

- Refactor postcss.config.js file to end in 'cjs' in order to get main page to appear on port 3000.
